cli/src/pcluster/models/cluster_config.py

```diff
@@ -200,15 +169,41 @@ def __init__(
         volume_id: str = None,
         raid: Raid = None,
     ):
-        Ebs.__init__(self, volume_type, iops, size, encrypted, kms_key_id, throughput)
+        super().__init__()
+        self.volume_type = Resource.init_param(volume_type, default=EBS_VOLUME_TYPE_DEFAULT)
+        self.iops = Resource.init_param(iops, default=EBS_VOLUME_TYPE_IOPS_DEFAULT.get(self.volume_type))
+        self.size = Resource.init_param(size, default=EBS_VOLUME_SIZE_DEFAULT)
+        self.encrypted = Resource.init_param(encrypted, default=False)
+        self.kms_key_id = Resource.init_param(kms_key_id)
+        self.throughput = Resource.init_param(throughput, default=125 if self.volume_type == "gp3" else None)
         self.mount_dir = mount_dir
         self.shared_storage_type = SharedStorageType.RAID if raid else SharedStorageType.EBS
         self.snapshot_id = Resource.init_param(snapshot_id)
         self.volume_id = Resource.init_param(volume_id)
         self.raid = raid
 
     def _validate(self):
-        super()._validate()
```

Comment:
It'd be better to keep `super()._validate()`. Because this class inherits a class that is not the base `Resource` class.

Reply:
I was thinking there's not _validate() in its super class for noe. At a second thought, adding it here maybe convenient if one day we add validate in its super class